integration-tests/utils/__init__.py

```diff
@@ -77,3 +77,17 @@ def prepare_args_for_connect(
         args.extend(["--server", server])
 
     return args
+
+
+def get_rhc_client_id():
+    cert_file = "/etc/pki/consumer/cert.pem"
+    try:
+        result = sh.openssl(f"x509 -in {cert_file} -subject -nocert".split()).strip()
+        if "CN" in result:
+            rhc_client_id = result.split()[1].split("=")[1]
+            return rhc_client_id
+        else:
+            return None
+    except sh.ErrorReturnCode as e:
+        print(f"An error occured: {e} ")
+        return None
```

Comment:
rather than this function, make explicit use of the `subman` fixture in `test_rhc_client_connect_e2e()`, and use `subman.uuid`

Reply:
thanks!